src/test/java/org/folio/rest/impl/CheckinReceivingApiTest.java

```diff
@@ -1031,45 +1031,6 @@ void testBindPiecesToTitleWithItem() {
 
   }
 
-  @Test
-  void testBindPiecesWithDifferentHoldingIdAndThrowError() {
```

Comment:
We can change this test to the case when location and holding exists or not exists

Reply:
Replaced test with holding creation scenario